File: p007/007.js
let counter = 0;
let prime = 0;
let num = 2;

while (counter < 10001) {
  debugger;
  if (checkPrime(num) == true) {
    counter++;
    prime = num;
    num++;
  } else num++;
}

console.log(prime);

function checkPrime(num) {
  // for (let i = 2; i < num; i++) {
  //   if (num % i === 0) return false;
  // }
  // return num > 1;
  let root = Math.sqrt(num);
  for (let i = 2; i <= root; i++) {
    if (num % i == 0) return false;
  }
  return true;
}

File: p007/007.py
import pdb
counter = 0
num = 2
prime = 0


def checkPrime(number):
    for i in range(2, number-1):
        if(number % i == 0):
            return False
        else:
            return True


while(counter < 6):
    pdb.set_trace()
    if(checkPrime(num) == True):
        counter += 1
        prime = num
        num += 1
    else:
        num += 1

print(prime)
